components/Button/index.js

```diff
@@ -36,7 +36,7 @@ const StyledButton = styled.button`
 `;
 
 const StyledAria = styled.a`
-  max-width: max-content;
+  max-width: 100%;
```

Comment:
@martinezpoa acho que não é aqui que temos que alterar. O botão já tem uma prop que controla o estado fullWidth ou não, porém essa prop não está correspondendo ao estilo 100% quando passada

Sugiro algo nessa linha:
```suggestion
  max-width: ${getWidthRule};
```